sdks/python/apache_beam/runners/worker/channel_factory.py

```diff
@@ -23,8 +23,24 @@
 
 class GRPCChannelFactory(grpc.StreamStreamClientInterceptor):
   DEFAULT_OPTIONS = [
-      ("grpc.keepalive_time_ms", 20000),
-      ("grpc.keepalive_timeout_ms", 300000),
+      # Default: 30000ms (30s), increased to 180s to reduce ping frequency
+      ("grpc.keepalive_time_ms", 180000),
+      # Default: 5000ms (5s), increased to 10 minutes for stability
```

Comment:
The default is 20 seconds

Channel Argument| Client|Server
----------------|-------|------
GRPC_ARG_KEEPALIVE_TIMEOUT_MS|20000 (20 seconds)|20000 (20 seconds)

https://github.com/grpc/grpc/blob/master/doc/keepalive.md

Reply:
Updated the comment.